README.md

```diff
@@ -9,7 +9,7 @@ Assuming that the [free 32 bit version of kdb+](http://kx.com/software-download.
 2. Download a zip of [this starter pack](https://github.com/AquaQAnalytics/TorQ-Finance-Starter-Pack/archive/master.zip)
 3. Unzip TorQ
 4. Unzip the starter pack over the top (this will replace some files)
-5. Run the appropriate starts script: start_torq_demo.bat for Windows, start_torq_demo.sh for Linux and start_torq_demo_osx.sh for Mac OS X.
+5. Run the appropriate starts script: start_torq_demo.bat for Windows, torq.sh for Linux and start_torq_demo_osx.sh for Mac OS X.
```

Comment:
Might need a little more detail here, can't just run `torq.sh` as you could with the old script, need to give it some args (although running it without any args should show the usage so maybe it's OK)